backend/src/services/collectionService.ts

```diff
@@ -552,4 +549,16 @@ export class CollectionService extends LoggerBase {
       widgets: [...widgets],
     }
   }
+
+  async findInsightsProjectsByName(name: string): Promise<any> {
+    const qx = SequelizeRepository.getQueryExecutor(this.options)
+    const result = await queryInsightsProjects(qx, {
+      filter: {
+        name: { eq: name },
```

Comment:
Can we improve this filter?
In addition to the name, find insights projects that have segmentId = null

Also, if there isn't an exact match by name can we also try to find with something like:
WHERE column_name ILIKE '%your_query%'